bundles/org.eclipse.osgi/container/src/org/eclipse/osgi/internal/framework/SystemBundleActivator.java

```diff
@@ -243,21 +239,26 @@ private void unintallSecurityManager() {
 		setSecurityManagner = null;
 	}
 
-	private void register(BundleContext context, Class<?> serviceClass, Object service, Dictionary<String, Object> properties) {
-		register(context, serviceClass.getName(), service, true, properties);
+	private ServiceRegistration<?> register(BundleContext context, Class<?> serviceClass, Object service,
```

Comment:
I don't see where the call sites for these `register` methods got changed to expect a registration returned.

Reply:
Line 210+211 in this change